src/schemas/AuthenticationRequestOpts.schema.ts

```diff
@@ -219,13 +219,27 @@ export const AuthenticationRequestOptsSchema = {
             }
           ]
         },
+        "credential_formats_supported": {
+          "anyOf": [
+            {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/CredentialType"
+              }
+            },
+            {
+              "$ref": "#/definitions/CredentialType"
+            }
+          ]
+        },
         "registrationBy": {
           "$ref": "#/definitions/RegistrationType"
         }
       },
       "required": [
-        "didMethodsSupported",
-        "registrationBy"
+        "credential_formats_supported",
```

Comment:
Although this is auto generated, I noticed that the didMethodsSupported is gone, which is odd

Reply:
A yupz make sense